### PR TITLE
Update UG, DG and PPP

### PR DESCRIPTION
**Pull request by trash-bin99**
UG:
- MInor bug fix

DG:
- Added diagrams for tag feature in DG
- Updated sections of the DG to be consistent with each other
- Fix grammar issues and missing commas

PPP:
- Added details regarding DG contributions

Please check for the grammar bits to make sure that the original meaning has not been changed, and if any bits from the DG should be removed/should not be removed.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1364,12 +1377,11 @@ testers are expected to do more *exploratory* testing.
...
-Character limit:
-1) Name - 0 to 50 characters
-2) Phone - 3 to 20 digits
-3) Email - max 100 characters (at least 2 for domain portion, before @ symbol)
+<div markdown="span" class="alert alert-primary">:information_source:
+**Character limits:** <br>
+1) Name - 0 to 50 characters <br>
+2) Phone - 3 to 20 digits <br>
+3) Email - max 100 characters (at least 2 for domain portion, before @ symbol) <br>
 4) Address - max 100 characters
```
Comment (Th-429B) [inline docs/DeveloperGuide.md:1385]: I think you missed out character limits for Remarks and Appointment Venue.

**Review by Th-429B — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (trash-bin99) [reply docs/DeveloperGuide.md:1385]: Character limits added for remark and tag.

**Comment by Th-429B** [comment]
LGTM

**Review by Th-429B — APPROVED** [review]
LGTM

**Pull request merged**